### DISCUSSION TITLE
Why am I getting "Deep Fry" on successive generations of my images?

### DISCUSSION THREAD

**Discussion by prairiefawkes** [Acly/krita-ai-diffusion#485] [Q&A] [answered]
I really love this tool because it makes composing sets much easier than others. But one thing I'm having trouble with is that I will throw down a very rough drawing to invoke a ~60% strength result that looks good. Then I'll transfer that result. What I'd like to do is start to overpaint that result. For instance overpainting a pair of glasses green. This works very well on InvokeAI's inpainting. However, I'm finding that the images that get returned from using an output as a base get successively deep fried. I've tried replicating ImgtoImg on other platforms to see if that happens, but it doesn't as far as I can tell. 

I've noticed that this happens when I'm doing Live generation. If I get a nice output and then copy the result to a new layer, it starts the deep fry cascade. I notice that I can take the deep-fried end output of that cascade and then pass it through a normal non-live "Generate" process and it gives me a nice looking result. 

Am I right in assuming that "strength" is simply "Denoising Strength"? Is there some principle of how SD works that I'm not getting?

What is the setting or issue that makes it so that a great output gets deep fried? 

Top image is my settings, and rest are progressive examples.

![image](https://github.com/Acly/krita-ai-diffusion/assets/156857407/249b0dca-c69e-4c6e-9c5b-39916b4afe9f)

![image](https://github.com/Acly/krita-ai-diffusion/assets/156857407/b2093d1f-18fa-4d03-98a4-061f1f73623d)
![image](https://github.com/Acly/krita-ai-diffusion/assets/156857407/4a83ae5c-4086-4553-8192-1ed35fb14601)
![image](https://github.com/Acly/krita-ai-diffusion/assets/156857407/699075a2-e6c5-43ca-9d65-a8561af1e2a0)
![image](https://github.com/Acly/krita-ai-diffusion/assets/156857407/bf1aa063-bd3d-4157-acec-11d40a6544ab)

**Comment by Acly** [comment, marked as answer]
It happens when you use the same seed.

There's a toggle in Interface Settings which enables generating a new seed automatically when you apply a result in Live painting, meant for workflow like yours. Or you can generate a new one manually when you need it.

**Reply by prairiefawkes** [reply]
Brilliant! Worked beautifully. Thank you so much for the fast response! And thank you for producing this library. It's wonderful.
